scripts/verify-package.sh

```diff
@@ -119,6 +119,12 @@ fi
 
 mkdir -p "test-reports/verify-package"
 
+# Must switch the node version back to v14.18.2 due to bug in v16 see https://github.com/npm/cli/pull/5894
+if ! setup_service node v14.18.2 &> /dev/null; then
+  echo "Failed to install node"
+  exit ${FAILED_SETUP}
+fi
```

Comment:
There was a bug in npm that caused `npm pack` to produce a pack report with a filename that contained the package scope instead of hyphens. Our verify package test validated against this file name and failed when using node v16.19.1, we have to explicitly set it back to v14.18.2 to get it to work.